install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
+   pilot_enabled: true
+   proxy_enabled: true
+   namespace: istio-system
+   ca_hub: default
+   ca_tag: default
+   ca_demo: false 
+   proxy_hub: default
+   proxy_tag: default
+   proxy_debug: false 
```

Comment:
One thing that isn't quite right is the helm chart should run out of the box correctly without modification.

The way this is managed currently with Istio is  https://github.com/istio/istio/blob/master/install/updateVersion.sh which reads the template files and merges them with  https://github.com/istio/istio/blob/master/istio.VERSION.  I think you may want to modify updateVersions.sh to modify the defaults in the helm chart or alternatively make updateVersions write out an override file or alternatively make a new tool which generates a helm versions override which can be passed via the CLI.

Reply:
@sdake thanks for pointing that out. If community agrees, I will extend updateVersion.sh so it would also update helm's values.yaml with current tags and hubs and other things.
@rshriram WDYT about extending updateVersion.sh ?

Reply:
I think that is a must if we are to keep this file upto date.

Reply:
done, now values.yaml gets updates with the same tags as the rest of Istio repo.